From 6207251ec22fd96a0121cd5d2837dd4d0ada3c3e Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Tue, 26 Oct 2021 22:11:58 +0200
Subject: [PATCH 2/2] fix build error: invalid operands to binary expression

---
 libmenu/matemenu-tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmenu/matemenu-tree.c b/libmenu/matemenu-tree.c
index 88d933d..1965982 100644
--- a/libmenu/matemenu-tree.c
+++ b/libmenu/matemenu-tree.c
@@ -1809,7 +1809,7 @@ gpointer
 matemenu_tree_item_ref (gpointer itemp)
 {
   MateMenuTreeItem* item = (MateMenuTreeItem*) itemp;
-  g_return_val_if_fail(item != NULL || *item != NULL, NULL);
+  g_return_val_if_fail(item != NULL, NULL);
   g_return_val_if_fail(item->refcount > 0, NULL);
 
   g_atomic_int_inc (&item->refcount);
-- 
2.36.1

